core/src/test/scala/unit/kafka/server/CreateTopicsRequestTest.scala

```diff
@@ -133,4 +140,35 @@ class CreateTopicsRequestTest extends AbstractCreateTopicsRequestTest {
     val response = sendCreateTopicRequest(req, notControllerSocketServer)
     assertEquals(1, response.errorCounts().get(Errors.NOT_CONTROLLER))
   }
+
+  @Test
+  def testOldestCreateTopicsRequest(): Unit = {
```

Comment:
Should we name this `testAllVersionsCreateTopicsRequest()` or `testCreateTopicsRequestVersions()`?

Reply:
`testCreateTopicsRequestVersions` sounds good to me.